moveit_configs_utils/default_configs/chomp_planning.yaml

```diff
@@ -0,0 +1,11 @@
+planning_plugin: chomp_interface/CHOMPPlanner
+enable_failure_recovery: true
+jiggle_fraction: 0.05
+request_adapters: "default_planner_request_adapters/AddTimeParameterization
+                   default_planner_request_adapters/ResolveConstraintFrames
+                   default_planner_request_adapters/FixWorkspaceBounds
+                   default_planner_request_adapters/FixStartStateBounds
+                   default_planner_request_adapters/FixStartStateCollision
+                   default_planner_request_adapters/FixStartStatePathConstraints"
```

Comment:
Super nit: [existing `*_planning.yaml`](https://github.com/ros-planning/moveit_resources/blob/ros2/panda_moveit_config/config/ompl_planning.yaml#L2-L8) and https://github.com/ros-planning/moveit_resources/pull/120/files uses `>-` for multiline string, could you please unify them for `*_planning.yaml` files?